test/util/util.go

```diff
@@ -255,6 +255,43 @@ func WaitForBindingCondition(client v1beta1servicecatalog.ServicecatalogV1beta1I
 	)
 }
 
+// WaitForBindingConditionLastSeenOfType waits for the status of the named
+// binding to contain a condition whose type and status matches the supplied one
+// but also returns back the last binding condition seen during pulling to help
+// identify bugs.
+func WaitForBindingConditionLastSeenOfType(client v1beta1servicecatalog.ServicecatalogV1beta1Interface, namespace, name string, condition v1beta1.ServiceBindingCondition) (*v1beta1.ServiceBindingCondition, error) {
```

Comment:
Can we just modify `WaitForBindingCondition` to return the last seen type? I don't see much value in having two separate functions.